### PR TITLE
Updated the how to install Solaar from the source

### PR DESCRIPTION
**Pull request by drupal-daffie**
Updated the how to install Solaar from the source

### CODE REVIEW

**Comment by drupal-daffie** [comment]
Comes from https://github.com/pwr-Solaar/Solaar/issues/526.

File: docs/installation.md [hunk 1]
```diff
@@ -29,14 +29,20 @@ installed.
 
 ### Installation
 
+The downloaded code from https://github.com/pwr-Solaar/Solaar only has the
+source files. The [byte code](https://en.wikipedia.org/wiki/Bytecode) that 
+is used by the Python interpreter must be build with the command: 
+`python ./setup.py build`. Then the Solaar program must be installed with the 
+command: `sudo python ./setup.py install`.
```
Comment (FFY00) [inline docs/installation.md]: Can we just put the commands?

Also, I would rather to recommend people to install with pip?

Comment (FFY00) [reply docs/installation.md]: So, `pip install .`.

Comment (drupal-daffie) [reply docs/installation.md]: It was 2 days ago that I for the first time installed a program on linux from the source. Can you give me the commands that is needed for a install with pip. I shall then update the pull request.

Comment (FFY00) [reply docs/installation.md]: The one I sent above, `pip install .`. Just make sure you run it in the top level of the repository.

**Comment by FFY00** [comment]
Closing this for now, feel free to re-open.

**Pull request closed without merging**